subsys/bluetooth/shell/bt.c

```diff
@@ -887,6 +899,26 @@ static int cmd_directed_adv(const struct shell *shell,
 
 	return 0;
 }
+
+static int cmd_adv_oob(const struct shell *shell, size_t argc, char *argv[])
```

Comment:
Why move OOB under peripheral?

Reply:
It was named with `_adv_` and its used in the context of testing in connections, right? Hence, moved under peripheral.

Reply:
It could also simply be used to fetch the advertising address. `cmd_oob` is also not under `CONFIG_BT_CONN`.
I would prefer that it was moved back. Was there any compilation problem with this?

Reply:
No, cmd_adv_info does not have the address, but if oob_local is under BT_CONN then fair enough.